bot/extensions/moderation/moderation.py

```diff
@@ -0,0 +1,50 @@
+from discord import Member
+import datetime
+from json import loads
+from turtle import title
+from discord.ext.commands.cooldowns import BucketType
+from discord.ext.commands import Cog, command, cooldown, has_permissions
+from discord import Embed
+
+
+class ModerationCog(Cog, name="moderation", description="Collection of administrative use commands."):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='kick', help="Allows a staff member to kick a user based on their behaviour.")
+    @has_permissions(kick_members=True)
+    async def kick(self, ctx, member: Member, reason="No reason"):
+            guild = ctx.guild
+            embed = Embed(title="Grace Moderation - KICK", description=f"{member.mention} has been kicked.", timestamp=datetime.datetime.utcnow())
+            embed.add_field(name="Reason: ", value=str(reason), inline=False)
+            await ctx.reply(embed=embed)
```

Comment:
As in Malassi's suggestion, can you also throw the Embed to the mod logs channel? The channel code should already be in the config class.